site/docs/master/httpproxy.md

```diff
@@ -715,7 +715,65 @@ spec:
           port: 80
 ```
 
-#### ExternalName
+#### Path Rewriting
+
+HTTPProxy supports rewriting the HTTP request URL path prior to delivering the request to the service.
```

Comment:
nit: Should maybe not use `service`? I understand what you mean, but maybe backend application or pod or something?

Reply:
i've always said something waffely like _backend service_

Reply:
Fixed.